lib/grape-swagger/doc_methods/data_type.rb

```diff
@@ -48,7 +48,7 @@ def parse_multi_type(raw_data_type)
         end
 
         def parse_entity_name(model)
-          if model.methods(false).include?(:entity_name)
+          if model.respond_to?(:entity_name)
```

Comment:
mmh, would `method_defined?` be the choice for it here?

Reply:
method_defined? is the old behavior, which only returns true if it is defined on *this* exact class.    (which is the old behavior).

What I'm wanting here is to allow a parent implemented entity_name to be used on child classes.